src/rules/apigw/APIGWRequestValidation.ts

```diff
@@ -27,6 +29,35 @@ export default Object.defineProperty(
           }
         }
       }
+      if (node.bodyS3Location) {
+        const assetOutdir = NagRules.resolveResourceFromInstrinsic(
+          node,
+          Stage.of(node)?.assetOutdir
+        );
+        const assetPath = NagRules.resolveResourceFromInstrinsic(
+          node,
+          node.getMetadata('aws:asset:path')
+        );
+        const specFile = yamlparse(
+          readFileSync(assetOutdir + '/' + assetPath, 'utf-8')
+        );
+        if ('x-amazon-apigateway-request-validators' in specFile) {
```

Comment:
The [docs](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-swagger-extensions-request-validator.html) mentions that this can be done on a per method basis as well instead of just globally.

Each method can also override the global config, that needs to be checked as well